pkg/schema/generate.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/alecthomas/jsonschema"
+	"io/ioutil"
+	"k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+const outputFile = "site/content/usage/20-schema.md"
+
+func main() {
+
+	var document strings.Builder
+	document.WriteString(`---
+title: Config file schema
+weight: 200
+---
+
+`)
+	document.WriteString("```yaml\n")
+
+	schema := jsonschema.Reflect(&api.Config{})
```

Comment:
Did you try it with our struct?

Reply:
yeah, I used the k8s one by mistake. Thanks!